build/release/release-procedure.adoc

```diff
@@ -0,0 +1,96 @@
+= Release Procedure
+
+== Projects and Artifacts
+Following sections describe what projects are part of the OptaPlanner release procedure and their resulting
+release artefacts.
```

Comment:
```suggestion
release artifacts.
```

First of all, let's be consistent. Second, let's fix this Brittish nonsense. :-)

Reply:
Thanks, this one escaped my attention.